staking/handler.go

```diff
@@ -91,7 +91,7 @@ func HandleMsgValidatorJoin(ctx sdk.Context, msg MsgValidatorJoin, k Keeper, con
 		ID:          validator.ID,
 		StartEpoch:  validator.StartEpoch,
 		EndEpoch:    validator.EndEpoch,
-		VotingPower: validator.VotingPower,
+		VotingPower: types.GetValidatorPower(validator.VotingPower),
```

Comment:
No need to do that since validator already has voting power.

Reply:
Yes, I found 2 functions, was planning on removing one.